connectivity/service/pom.xml

```diff
@@ -417,6 +401,14 @@ jmh-generator-annprocess). jmh-generator-annprocess overwrites the whole META-IN
                     </additionalOptions>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>3.0.0-M5</version>
```

Comment:
maven-plugin versions should be defined in the top-level pom.xml
or does this module explicitly need another maven-failsafe-plugin version than all the others?